btf/btf.go

```diff
@@ -321,6 +373,21 @@ func loadKernelSpec() (_ *Spec, fallback bool, _ error) {
 	return spec, true, err
 }
 
+func loadKernelModuleSpec(module string) (*Spec, error) {
+	base, _, err := kernelSpec()
+	if err != nil {
+		return nil, err
+	}
+
+	fh, err := os.Open("/sys/kernel/btf/" + module)
```

Comment:
This currently allows path traversal attacks with a module like `../foo`. Can we sanitize the name somehow? I'm guessing that Linux doesn't allow arbitrary names either.